<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TofCalcDlg</class>
 <widget class="QDialog" name="TofCalcDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Component Calculations</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Chopper</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="2" column="0">
        <widget class="QRadioButton" name="radioChopperOm">
         <property name="text">
          <string>Angular Frequency ω (rad/s):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioChopperR">
         <property name="text">
          <string>Chopper Radius r (cm):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editChopperOm">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editChopperL">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editChopperR">
         <property name="text">
          <string>25</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioChopperL">
         <property name="text">
          <string>Length of Window L (cm):</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="editChopperT"/>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="radioChopperT">
         <property name="text">
          <string>Burst Time Δt (μs, σ):</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="checkChopperCounterRot">
         <property name="text">
          <string>Pair of Counter-Rotating Discs</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Collimator</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioDivL">
         <property name="text">
          <string>Collimator Length L (cm):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="editDivL">
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioDivW">
         <property name="text">
          <string>Collimator Width w (cm):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="editDivW">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QRadioButton" name="radioDivAng">
         <property name="text">
          <string>Angular Acceptance Δφ (deg, σ):</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="editDivAng"/>
       </item>
       <item row="3" column="0" colspan="3">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>307</width>
           <height>93</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Velocity Selector</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioSelTh">
         <property name="text">
          <string>Blade Twist θ (deg):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioSelLam">
         <property name="text">
          <string>Neutron Wavelength λ (Å):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="radioSelL">
         <property name="text">
          <string>Selector Length L (cm):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editSelTh">
         <property name="text">
          <string>50</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="radioSelOm">
         <property name="text">
          <string>Angular Frequency ω (rad/s):</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="editSelOm"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editSelLam">
         <property name="text">
          <string>4.5</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editSelL">
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Mono. Focus</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editFocL2">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editFocL1">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Horizontal Curvature (m):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="editFocCurvV">
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="editFocCurvH">
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Mono. Angle 2θ_M (deg):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Length 1 (m):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Length 2 (m):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Vertical Curvature (m):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editFocTT">
         <property name="text">
          <string>90</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>radioChopperL</tabstop>
  <tabstop>editChopperL</tabstop>
  <tabstop>radioChopperR</tabstop>
  <tabstop>editChopperR</tabstop>
  <tabstop>radioChopperOm</tabstop>
  <tabstop>editChopperOm</tabstop>
  <tabstop>radioChopperT</tabstop>
  <tabstop>editChopperT</tabstop>
  <tabstop>checkChopperCounterRot</tabstop>
  <tabstop>radioDivL</tabstop>
  <tabstop>editDivL</tabstop>
  <tabstop>radioDivW</tabstop>
  <tabstop>editDivW</tabstop>
  <tabstop>radioDivAng</tabstop>
  <tabstop>editDivAng</tabstop>
  <tabstop>radioSelLam</tabstop>
  <tabstop>editSelLam</tabstop>
  <tabstop>radioSelTh</tabstop>
  <tabstop>editSelTh</tabstop>
  <tabstop>radioSelL</tabstop>
  <tabstop>editSelL</tabstop>
  <tabstop>radioSelOm</tabstop>
  <tabstop>editSelOm</tabstop>
  <tabstop>editFocL1</tabstop>
  <tabstop>editFocL2</tabstop>
  <tabstop>editFocTT</tabstop>
  <tabstop>editFocCurvV</tabstop>
  <tabstop>editFocCurvH</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TofCalcDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TofCalcDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
